Sources/AblyAssetTrackingInternal/AblyWrapper/DefaultAbly.swift

```diff
@@ -356,6 +370,36 @@ extension DefaultAbly: AblyPublisher {
         }
     }
     
+    public func sendResolution(
+        trackable: Trackable,
+        resolution: Resolution,
+        completion: ResultHandler<Void>?
+    ) {
+        guard let channel = channels[trackable.id] else {
```

Comment:
We should not send the resolution data via channel messages, we should be using channel's presence :sweat_smile: On Android, we also went with the channel messages approach, however, it generated a few issues and we've decided to switch to presence data (more details here https://github.com/ably/ably-asset-tracking-android/issues/638) :wink: